### ISSUE TITLE
EnderIO - Great Mod

### ISSUE THREAD

**Issue by mainesi** [StargateMC/Reboot#9]
This template should be followed when suggesting a mod be included in the reboot pack.

Read this before posting!

Note that some mods that do not require any modification whatsoever to be included in the server, such as RailCraft or IndustrialCraft 2.

If a mod or plugin needs to be modified to behave normally on the server (eg: It transports items wirelessly without penalty for distance), it must be open source (github available), for this to happen as we do not ban items and this will result in the mod being excluded from consideration.

Name: EnderIO
Type: Mod
Link to Github: https://github.com/SleepyTrousers/EnderIO
Link to License: http://creativecommons.org/publicdomain/zero/1.0/legalcode
License allows redistribution: unsure
Mod/Plugin version suggested: 
I have tested this mod in SinglePlayer: Yes
I have tested this mod/plugin on a test server running Thermos, or another server: Yes
Notes about tweaks needed for this mod or plugin to operate properly:

This mod holds a transportation element that is similar to thermal expansion tessaract cubes. But is also has one of the best plus sides, it allows cables to be placed within the same block and become entwined with each other

**Comment by PixelGamerX32** [comment]
I agree. This allows for multiple cables to fit in a one block space. More compact setups can be used.
